server/sonar-server/src/main/java/org/sonar/server/platform/ws/StatusAction.java

```diff
@@ -39,38 +35,36 @@
  */
 public class StatusAction implements SystemWsAction {
 
-  private static final Logger LOGGER = Loggers.get(StatusAction.class);
-
   private final Server server;
   private final DatabaseMigrationState migrationState;
   private final Platform platform;
-  private final DbClient dbClient;
   private final RestartFlagHolder restartFlagHolder;
 
   public StatusAction(Server server, DatabaseMigrationState migrationState,
     Platform platform, DbClient dbClient, RestartFlagHolder restartFlagHolder) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "dbClient". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)